### PR TITLE
Fix core forward pass (rope) during logits-test

### PR DESCRIPTION
**Pull request by dhia680**
This PR fixes the forward pass performed for the **logits-test** during HF conversion.
Indeed, the forward pass applied the standard rope, ignoring llama3 rope scaling (cf. `loader_core.py`).
It also allows loading correctly the `use_rope_scaling` argument.

### CODE REVIEW

**Review by AleHD — APPROVED** [review]
Looks nice!
